pkg/arvo/sys/zuse.hoon

```diff
@@ -7626,7 +7630,7 @@
       ::  #  constants
       ::
       ::  contract addresses
-      ++  contracts  mainnet-contracts
+      ++  contracts  ropsten-contracts
```

Comment:
gotta switch this back to mainnet

Reply:
On this point: I think we should avoid leaving testnet config lying around in the history at all.  I don't think there's ever a need to do this -- for testnet pills there's `make ropsten-pills`, and for convenience during development one can produce a testnet-configured Arvo via:

```
$ nix-build nix/pkgs -A arvo-ropsten --no-out-link
```

and that can be copied into a running testnet ship, etc.